js/piwik.js

```diff
@@ -5148,9 +5219,9 @@ if (typeof Piwik !== 'object') {
                  * @param string name (optional) The Event's object Name (a particular Movie name, or Song name, or File name...)
                  * @param float value (optional) The Event's value
                  */
-                trackEvent: function (category, action, name, value) {
+                trackEvent: function (category, action, name, value, callbacks) {
```

Comment:
There are methods like `trackLink` that currently expect `callback`. We'd need to make sure to use consistent naming and behaviour.